### PR TITLE
Better creation Example in docs & buff parameter in arrange

### PR DESCRIPTION
**Pull request by kolibril13**
Better creation example, the current one is a bit messy:
https://docs.manim.community/en/latest/reference/manim.animation.creation.html?highlight=creation#module-manim.animation.creation

### CODE REVIEW

**Comment by kolibril13** [comment]
https://manimce--825.org.readthedocs.build/en/825/reference/manim.animation.creation.html?highlight=creation#module-manim.animation.creation
new one

**Comment by leotrs** [comment]
Oh man I totally meant to fix that before merging the original PR. Thanks for picking up on that!

File: manim/animation/creation.py [hunk 1]
```diff
@@ -7,16 +7,40 @@
 
     class CreationModule(Scene):
         def construct(self):
-            texts = [Text('manim'), Text('manim')]
-            texts[0].shift(LEFT * 2 + UP)
-            texts[1].shift(RIGHT * 2 + UP)
+            s1 = Square()
+            s2 = Square().next_to(s1, RIGHT, buff=1.9)
+            s3 = Square().next_to(s2, RIGHT, buff=1.9)
+            s4 = Square().next_to(s3, RIGHT, buff=1.9)
```
Comment (leotrs) [inline manim/animation/creation.py:13]: Comment (non-blocking): I believe there is a better way of doing this by using `Mobject.arrange`, but I've never used it so I don't know if it actually works...

**Review by leotrs — APPROVED** [review]

File: manim/animation/creation.py (hunk 1 above)
Comment (kolibril13) [reply manim/animation/creation.py:13]: Uhh, nice!
I will have a look at this.
As nobody will ever see this code, I guess this won't be too important.

Comment (kolibril13) [reply manim/animation/creation.py:13]: Ok, I've actually changed it now.
It works like this:
```python
            class Example(Scene):
                def construct(self):
                    s1 = Square()
                    s2 = Square()
                    s3 = Square()
                    s4 = Square()
                    x = VGroup(s1, s2, s3, s4).set_x(0).arrange(buff=1.0)
                    self.add(x)
```

**Comment by kolibril13** [comment]
@leotrs :
I applied now your suggestion and further added a docstring to arrange.
Also, I added the parameter "buff" to arrange.

**Review by leotrs — APPROVED** [review]

**Review by huguesdevimeux — COMMENTED** [review]
SGTM, but I think a quick graphicval unit test would be appreciated :)

File: tests/test_graphical_units/test_geometry.py [hunk 2]
```diff
@@ -125,6 +125,13 @@ def construct(self):
         a = RoundedRectangle()
         self.play(Animation(a))
 
+class ArrangeTest(Scene):
```
Comment (kolibril13) [inline tests/test_graphical_units/test_geometry.py:128]: @huguesdevimeux : here is the test

Comment (huguesdevimeux) [reply tests/test_graphical_units/test_geometry.py:128]: perfect, thanks!

**Review by huguesdevimeux — APPROVED** [review]
SGTM

**Comment by kolibril13** [comment]
Nice!
I will merge this as soon as the test pass

File: manim/mobject/mobject.py [hunk 3]
```diff
@@ -1027,9 +1027,32 @@ def get_family(self):
...
+                    s1 = Square()
+                    s2 = Square()
+                    s3 = Square()
+                    s4 = Square()
+                    x = VGroup(s1, s2, s3, s4).set_x(0).arrange(buff=1.0)
+                    self.add(x)
+        """
         for m1, m2 in zip(self.submobjects, self.submobjects[1:]):
-            m2.next_to(m1, direction, **kwargs)
+            m2.next_to(m1, direction, buff, **kwargs)
```
Comment (behackl) [inline manim/mobject/mobject.py:1055]: In future instances, I'd like to suggest passing parameters like that as keyword parameters in library-internal code; it makes it much more stable and readable.

**Pull request merged**